package.json

```diff
@@ -18,8 +20,20 @@
   },
   "homepage": "https://github.com/dev1980/Testing-Practice-JS#readme",
   "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@babel/preset-env": "^7.8.4",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-preset": "^1.1.1",
+    "babel-preset-env": "^1.7.0",
```

Comment:
These devDeps are not required for this project, not sure if you are using them in any way.